Week1/homework.js

```diff
@@ -0,0 +1,113 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+});
+
+connection.connect();
+
+// creates the 'world' database
+const createDatabase = `CREATE DATABASE IF NOT EXISTS world`;
```

Comment:
You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql